libr/core/disasm.c

```diff
@@ -2755,6 +2755,22 @@ static void ds_instruction_mov_lea(RDisasmState *ds, int idx) {
 	case R_ANAL_OP_TYPE_CMOV:
 	case R_ANAL_OP_TYPE_MOV:
 		src = ds->analop.src[0];
+		if (ds->analop.ptr && !ds->analop.refptr) {
```

Comment:
Printing of strings for MOV-type instructions with no flag at referenced addr is done somewhere else in disasm.c ... are Dalvik strings being filtered out?